recipes/maxentscan/build.sh

```diff
@@ -0,0 +1,15 @@
+target=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+mkdir -p $target
+mkdir -p $PREFIX/bin
+
+#change hardcoded directory of splicemodels
+sed -i.bak "s@\"splicemodels/\"@\"$target/splicemodels/\"@" *.pl
+
+cp score3.pl $target/maxentscan_score3.pl
+cp score5.pl $target/maxentscan_score5.pl
+cp -r splicemodels $target
+
+chmod 0755 $target/*.pl
+ln -s $target/*.pl $PREFIX/bin
```

Comment:
The linked perl files are the actual scripts. They should be able to act as standalone executables. Is this the wrong way to do it? The linking and naming etc was heavily influenced (copied) by the recipe for variant-effect-predictor.